README.md

```diff
@@ -3,7 +3,10 @@
 ## Development
 
 1. Install [pipenv](https://pipenv.pypa.io/en/latest/)
+1. Create a lock file that allows the black beta release: `pipenv lock --pre`
 1. Install dependencies: `pipenv install`
+1. Install dependencies: `pipenv install --dev`
```

Comment:
Good catch. I think `pipenv install --dev` installs the default packages too though, so this can probably replace the previous `pipenv install` command.